### PR TITLE
add generic vpatch for freemarker ssti

### PR DESCRIPTION
**Pull request by seemanne**
This vulnerability has caused both a recent Confluence CVE (CVE-2023-22527) and the VMWare exploit we saw recently (CVE-2022-22954)

Comes with two seperate tests.

TODO:
- [x] Decide whether to add to appsec collection or make a new one for generic scenarios cc @buixor

### CODE REVIEW

**Comment by buixor** [comment]
We think we want a diff collection. We'll add libinjection detection and the upcoming ml model to this one.

File: appsec-rules/crowdsecurity/generic-freemarker-ssti.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        type: contains
+        value: 'freemarker.template.utility.execute'
+
+labels:
+   type: exploit
+   service: http
+   confidence: 2
+   spoofable: 0
+   behavior: "http:exploit"
+   label: "Generic FreeMarker SSTI Attempt"
```
Comment (AlteredCoder) [inline appsec-rules/crowdsecurity/generic-freemarker-ssti.yaml:20]: please remove "Attempt" from the label

File: appsec-configs/crowdsecurity/generic-rules.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+name: crowdsecurity/generic-rules
+default_remediation: ban
+#log_level: debug
+inband_rules:
+ - crowdsecurity/generic-*
+# inband_options:
```
Comment (AlteredCoder) [inline appsec-configs/crowdsecurity/generic-rules.yaml:6]: i think we can remove commented lines here

**Review by AlteredCoder — APPROVED** [review]

File: appsec-configs/crowdsecurity/generic-rules.yaml (hunk 2 above)
Comment (seemanne) [reply appsec-configs/crowdsecurity/generic-rules.yaml]: done

File: appsec-rules/crowdsecurity/generic-freemarker-ssti.yaml (hunk 1 above)
Comment (seemanne) [reply appsec-rules/crowdsecurity/generic-freemarker-ssti.yaml]: done

File: collections/crowdsecurity/appsec-generic-rules.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+name: crowdsecurity/appsec-generic-rules
+appsec-rules:
+  - crowdsecurity/generic-freemarker-ssti
```
[the remark this answers is not in the document]
Comment (seemanne) [reply collections/crowdsecurity/appsec-generic-rules.yaml:3]: Oh yes, good catch

**Pull request merged**